omod/src/main/java/org/openmrs/module/pihcore/page/controller/account/MyAccountPageController.java

```diff
@@ -3,122 +3,83 @@
  * Version 1.0 (the "License"); you may not use this file except in
  * compliance with the License. You may obtain a copy of the License at
  * http://license.openmrs.org
- *
+ * <p>
```

Comment:
Random?  @mseaton

Reply:
@mogoodrich - intellij gives me little yellow warnings about blank lines in javadoc, and suggests adding <p> tags, which gets rid of the little warnings.  I'm guessing it's the correct format.